### PR TITLE
Floodscan

### PR DESCRIPTION
**Pull request by turnerm**
Processing of floodscan data

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,10 @@
-
+anticipy==1.0.1
```
Comment (zackarno) [inline requirements.txt:1]: to get this to load i had to change to `ocha-anticipy` and remove the `==` pin. Should be fine to adjust in more up to date branch?

File: src/utils.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+logger = logging.getLogger(__name__)
+
+
+def load_codab(admin_level=0):
+    codab = CodAB(constants.country_config)
+    # codab.download()
+    return codab.load(admin_level=admin_level)
+
+
+# question: should this preprocessing go here or in datasource_extensions?
```
Comment (zackarno) [inline src/utils.py:20]: no idea tbh.  I'm probably the only one at this point, but i'd find a little mini session on how we use `datasouce_extensions` and even `ocha-anticipy` super helpful

Comment (turnerm) [reply src/utils.py:20]: I think this is an old question from Tinka that I forgot to take out, I believe I fixed it in the next branch. But happy to show you more about how the DataSources class works!

**Comment by turnerm** [comment]
Merging as-is and will fix reqs upstream

**Pull request merged**